src/System.Runtime.CompilerServices.VisualC/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
+      uap-Windows_NT;
```

Comment:
>I added this to match the source project 

Not necessarily. For things like this which are part of netstandard and doesn't have any unique APIs above netstandard you should make the test project have a netstandard configuration, but the src/ref projects cannot have a netstandard configuration because they are part of netstandard. 

I suggest you just have a netstandard configuration here.